integration_tests/src/main/python/date_time_test.py

```diff
@@ -573,9 +573,11 @@ def test_timestamp_seconds_long_overflow():
         lambda spark : unary_op_df(spark, long_gen).selectExpr("timestamp_seconds(a)").collect(),
         conf={},
         error_message='long overflow')
-    
-@pytest.mark.parametrize('data_gen', [DecimalGen(7, 7), DecimalGen(20, 7)], ids=idfn)
+
+# Make sure every decimal value in test data is rounding necessary by set full_precision=True to
+# avoid leading and trailing zeros
```

Comment:
typo: ```tailing```

Reply:
Seems `trailing zeros` is correct, I made a typo in PR title.